client/me/purchases/manage-purchase/index.jsx

```diff
@@ -646,10 +647,42 @@ class ManagePurchase extends Component {
 					</>
 				);
 			}
-
 			return description;
 		}
 
+		if ( isDIFMProduct( purchase ) ) {
+			const productDetails = productsList[ WPCOM_DIFM_LITE ];
+			const [ tier0 ] = productDetails.price_tier_list;
+			const { maximum_units: noOfBaselinePagesIncluded } = tier0;
```

Comment:
Nit: Maybe rename to `numberOfIncludedPages`? Asking because the `baseline` term might be confusing here.